src/reporter.ts

```diff
@@ -110,12 +115,18 @@ function getCSVFormat(
       type: Membership.PENDING_INVITE
     }
   })
-  const allMembers = members.concat(outsideCollaborators).concat(pendingInviteMembers)
+
+  const allMembers = members
+    .concat(outsideCollaborators)
+    .concat(pendingInviteMembers)
+    .sort((a, b) => a.login.localeCompare(b.login))
+
   const membersContent = [
     ['Login', 'Emails', 'Orgs', 'Membership', 'Created At'],
     ...allMembers.map(item => [
       item.login,
-      item.emails.join(','),
+      // Make sure the email only appears once
+      Array.from(new Set(item.emails)).join(','),
```

Comment:
```suggestion
      [...new Set(item.emails)].join(','),
```

Might be the easier syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#remove_duplicate_elements_from_the_array